opentelemetry-otlp/src/span.rs

```diff
@@ -159,4 +159,12 @@ impl opentelemetry_sdk::trace::SpanExporter for SpanExporter {
             SupportedTransportClient::Http(client) => client.set_resource(resource),
         }
     }
+    fn shutdown(&self) -> OTelSdkResult {
+        match &self.client {
```

Comment:
additionally to @scottgerring suggestion you also need to make it mutable here
```suggestion
        match &mut self.client {
```

Reply:
just like other signals, have made the change in the trait to not requiring mutable self